app/workgroup/services/workgroupStateService.js

```diff
@@ -172,6 +172,25 @@ workgroupApp.service('workgroupStateService', function ($rootScope, Role, Tag, L
 					return roles;
 			}
 		},
+		_uiReducers: function (action, ui) {
+			var scope = this;
+
+			switch (action.type) {
+				case INIT_WORKGROUP:
+					ui = {
+						addUserInProgress: false
+					};
+					return ui;
+				case BEGIN_ADD_USER:
```

Comment:
If the purpose of these two events is to turn a flag on and off, let's make the names symmetric. ADD_USER_PENDING, ADD_USER_COMPLETED, for example.